3303Elevator/src/app/ElevatorSubsystem/Elevator/Elevator.java

```diff
@@ -1,137 +1,238 @@
 package app.ElevatorSubsystem.Elevator;
 
 import app.Scheduler.TimeManagementSystem;
+import app.Logger;
 import app.ElevatorSubsystem.Direction.Direction;
 import app.ElevatorSubsystem.StateMachine.*;
 import java.util.ArrayList;
 /**
  * SYSC 3303, Final Project
  * Elevator.java
  * Purpose: contains the state of the elevator
- * 
+ *
  * @author Ben Kittilsen
  * */
 public class Elevator{
 
+	private static int count = 1;
+	private final int id;
+
 	private ElevatorDoor door;
 	private int currentFloor;
 	private ElevatorStateMachine state;
 	private TimeManagementSystem tms;
-	
+	private Logger logger;
+
 	/**
 	 * Maximum number of floors
 	 */
 	private int maxFloorCount;
 
-	public Elevator(int maxFloorCount, TimeManagementSystem tms) {
+	public Elevator(int maxFloorCount,Logger logger, TimeManagementSystem tms) {
 		this.maxFloorCount = maxFloorCount;
 		this.currentFloor = 1;
```

Comment:
Logger comms will be done via UDP packets instead of a direct reference to the logger obj